Bottom: 4402622a693297c8313b94da5409e0a1c50e1c9b
Top:    64998239839652bb9d3f74debd78390769c9b68c
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-16 20:56:14 +0800

Refresh of vhost_net-rx-batching

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 8344c04..a2b7391 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -103,6 +103,7 @@ struct vhost_net_virtqueue {
 	 * Protected by vq mutex. Writers must also take device mutex. */
 	struct vhost_net_ubuf_ref *ubufs;
 	struct skb_array *rx_array;
+	/* FIXME: rename */
 	void *skbs[VHOST_RX_BATCH];
 	int nskbs;
 	int skb_index;
